src/Template/Common/ComposerJsonTemplate.php

```diff
@@ -43,8 +43,15 @@ public function __toString(): string
                     "emulgeator/enum": "^1.0",
                     "emulgeator/array-to-class-mapper": "^0.1",
                     "psr/http-message": "^1.0",
+                    "psr/http-message-implementation": "^1.0",
                     "psr/http-client": "^1.0",
-                    "psr/http-factory": "^1.0"
+                    "psr/http-client-implementation": "^1.0",
+                    "psr/http-factory": "^1.0",
+                    "psr/http-factory-implementation": "^1.0"
+                },
+                "require-dev": {
+                    "guzzlehttp/guzzle": "^7.4",
```

Comment:
To allow doing an install, and testing for example, we should provide an example implementation